### PR TITLE
chore: implement landing page changes

### PR DESCRIPTION
**Pull request by tobySolutions**
In this PR, changes made were to fix issue #107:

- Change the Studio text on the top left to say Akeru. Keep the logo and its animation in place.: For this, I came to the resolution that it would be better to have the `Akeru` text be an SVG that we extract from a Figma design so we can keep the  logo; because, the "Studio" is an SVG and not a text. cc: @GuiBibeau 

-  Change the Contact Us button to become a [link to the GitHub repo](https://github.com/AkeruAI). Text says Star on Github!.
- Hide the burger menu icon. Keep the menu in place, but for now, we should not be able to access it. 🍔
- The Hero text became `Edge AI Subnet by OSS Capital.`
- Hero subtext became: `Join the waiting list.`
- Under the subtext, I brought in the email address field that is currently in the footer.  I also brought in the logic to register to the waiting list that we are currently using on the old landing page.
- Success state of the waiting list becomes a checkmark where there is currently an arrow.

### CODE REVIEW

File: packages/studio-ts/src/components/WaitlistForm.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    >
+      <path d="M20.285 2l-11.285 11.567-5.286-5.011-3.714 3.716 9 8.728 15-15.285z" />
+    </svg>
+  )
+}
+
+export function WaitlistForm() {
+  const [state, formAction] = useFormState(handleWaitlistSubmit, initialState)
+
+  console.log(state?.submitted)
```
Comment (TommieTechie) [inline packages/studio-ts/src/components/WaitlistForm.tsx:42]: Can this be removed?

**Review by TommieTechie — APPROVED** [review]
There are minor cleanups we might want to consider before merging.

File: packages/studio-ts/src/app/server.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(rawFormData),
+    })
+
+    if (response.ok) {
+      console.log('Email address sent successfully!!!')
```
[the remark this answers is not in the document]
Comment (tobySolutions) [reply packages/studio-ts/src/app/server.tsx:22]: Oh yes, I was debugging with it; I can remove it as we do not need it anymore.

File: packages/studio-ts/src/app/server.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      return { message: 'Email address sent successfully!', submitted: true }
+    } else {
+      console.error(
```
[the remark this answers is not in the document]
Comment (tobySolutions) [reply packages/studio-ts/src/app/server.tsx:25]: Done!

File: packages/studio-ts/src/app/server.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        'Failed to send email address. Status code:',
+        response.status,
+      )
+    }
+  } catch (error) {
+    console.error('Error:', error)
```
[the remark this answers is not in the document]
Comment (tobySolutions) [reply packages/studio-ts/src/app/server.tsx:31]: Done!

File: packages/studio-ts/src/components/WaitlistForm.tsx (hunk 1 above)
Comment (tobySolutions) [reply packages/studio-ts/src/components/WaitlistForm.tsx:42]: At the moment, no we do not have to keep it 😁 . I just removed it!

File: packages/studio-ts/src/app/server.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    return { message: 'Email address not sent', submitted: false}
+  }
+  //   redirect('/success')
```
[the remark this answers is not in the document]
Comment (tobySolutions) [reply packages/studio-ts/src/app/server.tsx:34]: At the moment, no we do not have to keep it 😁 . I just removed it!

File: packages/studio-ts/src/components/WaitlistForm.tsx (hunk 1 above)
Comment (tobySolutions) [reply packages/studio-ts/src/components/WaitlistForm.tsx:42]: Done!

File: packages/studio-ts/src/components/RootLayout.tsx [hunk 6]
```diff
@@ -81,10 +81,10 @@ function Header({
           />
         </Link>
         <div className="flex items-center gap-x-8">
-          <Button href="/contact" invert={invert}>
-            Contact us
+          <Button href="https://github.com/AkeruAI" invert={invert}>
```
Comment (GuiBibeau) [inline packages/studio-ts/src/components/RootLayout.tsx:84]: This links to the org, let's link to the repo https://github.com/AkeruAI/akeru

Comment (tobySolutions) [reply packages/studio-ts/src/components/RootLayout.tsx:84]: Okay, awesome!

**Review by GuiBibeau — APPROVED** [review]

**Pull request merged**